pkg/patterns/declarative/pkg/applier/direct.go

```diff
@@ -39,10 +40,19 @@ func (d *DirectApplier) Apply(ctx context.Context, opt ApplierOptions) error {
 		RESTConfig: opt.RESTConfig,
 	}
 	b := resource.NewBuilder(restClientGetter)
+	f := cmdutil.NewFactory(&genericclioptions.ConfigFlags{})
 
 	if opt.Validate {
 		// This potentially causes redundant work, but validation isn't the common path
-		v, err := cmdutil.NewFactory(&genericclioptions.ConfigFlags{}).Validator(true)
+		// v, err := cmdutil.NewFactory(&genericclioptions.ConfigFlags{}).Validator(true)
+
```

Comment:
Nit: I would probably just remove this line completely rather than commenting it out.